### PR TITLE
[Fixes #238] Update to advanced installation for geonode-project

### LINKED ISSUE #238: Update and review GeoNode Project section for master
- review the steps, if they're still valid or need to be updated
- update the referenced versions of GeoNode and, in case, of any dependency
- update to Ubuntu 20.04

https://docs.geonode.org/en/master/install/advanced/project/index.html#

### PR DESCRIPTION
**Pull request by jkariukidev**
ref #238

### CODE REVIEW

File: install/advanced/project/index.rst [hunk 1]
```diff
@@ -144,16 +152,16 @@ Make an instance out of the ``Django Template``
 
   # Install the Python packages
   cd /opt/geonode_custom/my_geonode
-  pip install -r requirements.txt --upgrade --no-cache --no-cache-dir
-  pip install -e . --upgrade
+  pip install -r src/requirements.txt --upgrade --sno-cache-dir
```
Comment (giohappy) [inline install/advanced/project/index.rst:155]: @jkariukidev what is this option? I guess you cut the original params by error

File: install/advanced/project/index.rst [hunk 2]
```diff
@@ -68,15 +74,17 @@ First, we are going to install all the **system packages** needed for the GeoNod
...
   sudo apt install -y git unzip gcc zlib1g-dev libgeos-dev libproj-dev
   sudo apt install -y sqlite3 spatialite-bin libsqlite3-mod-spatialite
 
   # Install Openjdk
   sudo -i apt update
   sudo apt install openjdk-8-jdk-headless default-jdk-headless -y
-  sudo update-java-alternatives --jre-headless --jre --set java-1.8.0-openjdk-amd64
+
+  # Remember to select the correct java version /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+  sudo update-alternatives --config java
```
Comment (giohappy) [inline install/advanced/project/index.rst:87]: does this step need user interaction?

Comment (jkariukidev) [reply install/advanced/project/index.rst:87]: Yes.

File: install/advanced/project/index.rst (hunk 1 above)
Comment (jkariukidev) [reply install/advanced/project/index.rst]: Was introduced during refactoring by accident. I have corrected it.

File: install/advanced/project/index.rst (hunk 2 above)
Comment (jkariukidev) [reply install/advanced/project/index.rst]: Reverted the command back to previous as it works despite the error thrown in terminal.

**Comment by giohappy** [comment]
@josephjosedev the Docker section wasn't updated.

**Pull request merged**